pytorch_lightning/trainer/model_hooks.py

```diff
@@ -22,13 +23,14 @@ class TrainerModelHooksMixin(ABC):
 
     lightning_module: LightningModule
 
-    def is_function_implemented(self, f_name, model=None):
+    def is_function_implemented(self, f_name: str, model: Optional[LightningModule] = None) -> bool:
+        # note: currently unused - kept as it is public
```

Comment:
if it's unused we need to add pragma no cover to exclude it from coverage
or test it anyway xD

Reply:
I initially removed it: https://github.com/PyTorchLightning/pytorch-lightning/pull/6386#discussion_r588990207

What do you think is the best option?

cc: @Borda